### PR TITLE
Feature/ch882/build component heading section

### PR DESCRIPTION
**Pull request by SamMed1**

### CODE REVIEW

**Comment by SamMed1** [comment]
Thanks for the comment @joesb 

Good question in regards to using rem / em units, it's something that I've considered, and am open to discussion around the best way to implement the designs.

My thoughts on this at the moment relate to my approach to the site build. For code reusability and for future proofing (whereby the site can change to/from different platforms) , I am using a block-based approach to the build, where the different content blocks (heading section, content section, image section etc) can be recycled and used wherever in the site, with the same styling applied. The idea is that there will be minimal nesting of styles due to styling being element-specific (as far as possible), and that using rem units, relative to the root fallback px size, would suffice.

**Comment by joesb** [comment]
@SamMed1 — that sounds like a good rationale on rem/em usage and your block-based approach, thinking of future usage scenarios in the process of building this. Very happy to read that stuff — nice one.

@lewisnyman is there something in particular that needs to be done to trigger Netlify checks and deploy previews, something that's not happened in this PR? I've tried comparing the master branch files and file structure with this one, and I can't work out if there's a particular file that's missing, but maybe it's just some snow blindness on my part.

File: _includes/assets/sass/_variables.scss [hunk 1]
```diff
@@ -40,26 +40,20 @@ $base-font-size: 19px;
 $h1-font-size-mobile: 1.158rem;
 $h1-font-size: 1.526rem;
 
-$h2-font-size-mobile: 0rem;
-$h2-font-size: 0rem;
+$h2-font-size-mobile: 1rem;
+$h2-font-size: 1.211rem;
```
Comment (lewisnyman) [inline _includes/assets/sass/_variables.scss]: @SamMed1 What do you think about reusing the px-to-rem ultility from govuk frontend to make this easy to read? https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/tools/_px-to-rem.scss

Comment (SamMed1) [reply _includes/assets/sass/_variables.scss]: @lewisnyman - Apologies for the delayed response. I wanted to test this before giving my thoughts. I think this is a great idea, and will go a long way in making values more readable.

I would like to roll this out across the whole site. As all values so far have been defined in rem, I suggest we merge this branch in, and then create a new branch for implementing this, if that works with you?

Comment (SamMed1) [reply _includes/assets/sass/_variables.scss]: @lewisnyman - I've just seen that you are on holiday today. My strategy is going to be merging this in, then rolling out the above on a separate branch, just so we can keep a clear record of changes.

**Pull request merged**